Description: Resolves issue where YAML module is reported missing when .gemrc is gone.
Author: Clint Byrum <clint@ubuntu.com>
Bug: http://rubyforge.org//tracker/index.php?func=detail&aid=28582&group_id=126&atid=575
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libgems-ruby/+bug/643768
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=597554

Index: libgems-ruby-1.3.7/lib/rubygems/specification.rb
===================================================================
--- libgems-ruby-1.3.7.orig/lib/rubygems/specification.rb	2010-11-02 15:10:57.385372002 -0700
+++ libgems-ruby-1.3.7/lib/rubygems/specification.rb	2010-11-02 15:11:08.175372000 -0700
@@ -4,6 +4,8 @@
 # See LICENSE.txt for permissions.
 #++
 
+require 'yaml'
+
 require 'rubygems/version'
 require 'rubygems/requirement'
 require 'rubygems/platform'
